deployment/scripts/codeship_aws

```diff
@@ -13,6 +13,28 @@ SCRIPT_NAME=${1:?'You need to provide the action as the first parameter'}
 SCRIPT_PATH=$DIR/codeship_aws_$SCRIPT_NAME
 
 if [ -x $SCRIPT_PATH ]; then
+  if [ ! -z "${CODESHIP_VIRTUALENV}" ]; then
+    #deactivate old virtualenv if one was activated and store in variable
+    if [ "" != "${VIRTUAL_ENV}" ]; then
+      OLD_VIRTUAL_ENV=${VIRTUAL_ENV}
+      if [ "$(type -t deactivate)" == "function" ]; then
+        deactivate
+      fi
+    fi
+    source "${CODESHIP_VIRTUALENV}/bin/activate"
+  fi
+  aws "$@"
```

Comment:
I think this should be `$SCRIPT_PATH "${@:2}"` - otherwise we'd always call `aws` plainly, es we do in the case when `$SCRIPT_PATH` is not executable/set.